server/src/main/java/org/opensearch/search/aggregations/AggregationBuilder.java

```diff
@@ -100,11 +109,13 @@ public String getName() {
     public abstract AggregationBuilder subAggregation(PipelineAggregationBuilder aggregation);
 
     /** Return the configured set of subaggregations **/
+    @JsonProperty("subAggregations")
```

Comment:
I didn't think this was necessary if the property name exactly matched the getter syntax. (`getS` ... `s` and the rest is the same).  Same comment below for pipelines.

Reply:
Registering the modules in SDKClient helped to remove the JsonProperty for OpenSearch and in AD Extension as well.